main.go

```diff
@@ -16,6 +19,15 @@ import (
 
 var keptnOptions = keptn.KeptnOpts{}
 
+type gracefulShutdownKeyType struct{}
+
+var gracefulShutdownKey = gracefulShutdownKeyType{}
+
+// Opaque key type used for graceful shutdown context value
+type keptnQuitType struct{}
+
+var testRunnerQuit = keptnQuitType{}
```

Comment:
Not sure if we should call it `testRunnerQuit`. How about `serviceRunnerQuit`?

Reply:
Yes makes sense.